force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -288,8 +374,15 @@ public with sharing class FlowControlLoopsCollections {
     public static Decimal countMoney() {
         // The list of money in the wallet, represented in dollars and cents
         List<Decimal> moneyInWallet = new List<Decimal>{0.50, 10, 3.84, 24.60, 9.08, 50, 4.90}; //DO NOT CHANGE
-
-        return null; // Replace null with the variable you used to store the result
+        Decimal totalValue = 0;
+        for(Integer i = 0; i < moneyInWallet.size(); i++){
+            totalvalue += moneyInWallet[i]; //totalValue (Starting at zero plus the values in the index, which is moneyInWallet[i])
+            
+            If (totalValue > 40){//the loop should break if it exceeds 40 dollars.
```

Comment:
Try adding a variable for the number 40 so it's not a magic number  https://en.wikipedia.org/wiki/Magic_number_(programming)